src/ML-IAP/mliap_so3.cpp

```diff
@@ -733,28 +733,27 @@ void MLIAP_SO3::get_sbes_array(int nlocal, int *numneighs, double **rij, int lma
         sa = sinh(rb) / rb;
         sb = (cosh(rb) - sa) / rb;
 
-        m_sbes_array[gindex + (i - 1) * mindex + 0] = sa;
-        m_sbes_array[gindex + (i - 1) * mindex + 1] = sb;
+        m_sbes_array[gindex + i1mindex + 0] = sa;
+        m_sbes_array[gindex + i1mindex + 1] = sb;
 
         for (j = 2; j < lmax + 1; j++)
-          m_sbes_array[gindex + (i - 1) * mindex + j] =
-              m_sbes_array[gindex + (i - 1) * mindex + j - 2] -
-              (2 * j - 1) / rb * m_sbes_array[gindex + (i - 1) * mindex + j - 1];
+          m_sbes_array[gindex + i1mindex + j] = m_sbes_array[gindex + i1mindex + j - 2] -
+              (2 * j - 1) / rb * m_sbes_array[gindex + i1mindex + j - 1];
 
         exts = m_sbes_array[gindex + (i - 1) * mindex + j - 2] -
-            (2 * j - 1) / rb * m_sbes_array[gindex + (i - 1) * mindex + j - 1];
+            (2 * j - 1) / rb * m_sbes_array[gindex + i1mindex + j - 1];
 
-        m_sbes_darray[gindex + (i - 1) * mindex + 0] = sb;
+        m_sbes_darray[gindex + i1mindex + 0] = sb;
 
         for (j = 1; j < lmax; j++)
-          m_sbes_darray[gindex + (i - 1) * mindex + j] = xi *
-              (j * m_sbes_array[gindex + (i - 1) * mindex + j - 1] +
-               (j + 1) * m_sbes_array[gindex + (i - 1) * mindex + j + 1]) /
+          m_sbes_darray[gindex + i1mindex + j] = xi *
+              (j * m_sbes_array[gindex + i1mindex + j - 1] +
+               (j + 1) * m_sbes_array[gindex + i1mindex + j + 1]) /
               (2 * j + 1);
 
-        m_sbes_darray[gindex + (i - 1) * mindex + j] = xi *
-            (j * m_sbes_array[gindex + (i - 1) * mindex + j - 1] + (j + 1) * exts) / (2 * j + 1);
-        m_sbes_darray[gindex + (i - 1) * mindex + 0] = xi * sb;
+        m_sbes_darray[gindex + (i - 1) * mindex + j] =
```

Comment:
Could replace (i-1)*mindex with m1index here too?

Reply:
Thanks. I found this and another case that must have slipped through the previous find+replace search.